frontend/src/app/main/ui/ds/layout/tab_switcher.cljs

```diff
@@ -184,14 +186,20 @@
       (mf/set-ref-val! tabs-ref tabs))
 
     [:> :article props
-     [:div {:class (stl/css :padding-wrapper)}
-      [:> tab-nav* {:button-position action-button-position
-                    :action-button action-button
-                    :tabs tabs
-                    :on-ref on-ref
-                    :selected selected
-                    :on-key-down on-key-down
-                    :on-click on-click}]]
+     [:div {:class (stl/css :tabs-nav-container)}
```

Comment:
This new container breaks the layout a little bit with the paddings.

Before
![Screenshot from 2025-01-02 10-33-26](https://github.com/user-attachments/assets/f992b098-897b-412d-9b75-4cc0659c780b)

After
![Screenshot from 2025-01-02 10-33-18](https://github.com/user-attachments/assets/97487cc9-4891-4e1d-8331-9775ac4f0fde)